src/sagemaker_inference/transformer.py

```diff
@@ -75,21 +73,22 @@ def __init__(self, default_inference_handler=None):
         self._output_fn = None
 
     @staticmethod
-    def handle_error(context, inference_exception):
+    def handle_error(context, inference_exception, tb):
```

Comment:
nit: Should we rename this variable from `tb` to `trace` or something similar to make it clearer what it is (and since the `traceback` name is taken by the module)?